crypto/ghash-generic.c

```diff
@@ -61,8 +61,7 @@ static int ghash_setkey(struct crypto_shash *tfm,
 	if (keylen != GHASH_BLOCK_SIZE)
 		return -EINVAL;
 
-	if (ctx->gf128)
-		gf128mul_free_4k(ctx->gf128);
+	gf128mul_free_4k(ctx->gf128);
```

Comment:
This cleanup in ghash-generic.c should be a separate patch.  How about sending it out now as its own patch?  It doesn't need to wait for anything else.